applications/Tellor.md

```diff
@@ -0,0 +1,231 @@
+
+
+Github profiles of our dev team:
+
+- https://github.com/themandalore - Nick Fett CTO
+- https://github.com/brendaloya - Brenda Loya CEO 
+- https://github.com/tkernell  -   Integrations Engineer
+- https://github.com/orgs/tellor-io/people/akremstudy - Developer
+- https://github.com/tallywiesenberg - Developer
+- https://github.com/oraclown - Developer
+
+### Team LinkedIn Profiles (if available)
+
+- https://www.linkedin.com/in/brendaloya/ 
+- https://www.linkedin.com/in/nicholas-fett/
+- https://www.linkedin.com/in/zemrose
+- https://www.linkedin.com/in/timothy-kernell-27b213102/
+- https://www.linkedin.com/in/tally-wiesenberg/ 
+- https://www.linkedin.com/in/owenburton22/
+
+## Development Status :open_book:
+
+The smart contract structure for Tellor is finished for most EVM chains, but will need specific changes. Initial design discussions have been had with Robin, Frank Bell, and Guatam and we have specifications and even some base work done around the design.  
+
+- Medium Articles - https://medium.com/@tellor 
+- Research - ​​https://vc.bridgew.edu/cgi/viewcontent.cgi?article=1037&context=ijcic 
+- Smart Contract repository - https://github.com/tellor-io/tellorFlex 
+- Reporter (Node) repository - https://github.com/tellor-io/telliot-feeds  
+
+## Development Roadmap :nut_and_bolt:
+
+| Milestone | Description                                                                      | Stack                 | Resources | Funding |
+| --------- | -------------------------------------------------------------------------------- | --------------------- | --------- | ------- |
+| 1         | Develop and launch Tellor core contracts on an EVM parachain                     | Solidity / javascript | 2 devs    | 20,000  |
+| 2         | Create and test oracle pallet and complete documentation with usage examples     | Rust / Solidity       | 2 devs    | 20,000  |
+
+
+
+
+
+### Overview
+
+- **Total Estimated Duration:** 4 months
+- **Full-Time Equivalent (FTE):**  2 
+- **Total Costs:** 40,000 USD
+
+
+### Milestone 1 — Launch Tellor core contracts on an EVM parachain
+
+
+- **Estimated duration:** 2 month
+- **FTE:**  2
+- **Costs:** 20,000 USD
+
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| **0a.** | License |  MIT|
+| **0b.** | Documentation | We will provide both **inline documentation** of the code and a basic **tutorial**. |
+| **0c.** | Testing and Testing Guide | Core functions will be fully covered by comprehensive unit tests to ensure functionality and robustness. In the guide, we will describe how to run these tests. |
+| **0d.** | Docker | We will provide a Dockerfile(s) that can be used to test all the functionality delivered with this milestone. |
+| 1 | Develop Controller contracts | We will provide a set of solidity smart contracts with the functionality described above  |
+| 2 | Develop Parachain integration contract| We will provide an integration contract |
+| 3 | Local testing| |
```

Comment:
As @dsm-w3f said, this still requires a specification.

Reply:
You are correct deliverable 0c should cover the testing so I’ve removed deliverable 3 from milestone 1.